robot/client/client_test.go

```diff
@@ -507,10 +510,15 @@ func TestClientRefresh(t *testing.T) {
 		logger,
 		WithRefreshEvery(dur),
 	)
+	test.That(t, callCountSubtypes, test.ShouldEqual, 1)
+	test.That(t, callCountNames, test.ShouldEqual, 1)
 	test.That(t, err, test.ShouldBeNil)
 	<-calledEnough
 	test.That(t, time.Since(start), test.ShouldBeGreaterThanOrEqualTo, 5*dur)
 	test.That(t, time.Since(start), test.ShouldBeLessThanOrEqualTo, 10*dur)
+	// during this duration, the client refreshed 4 additional times
+	test.That(t, callCountSubtypes, test.ShouldEqual, 5)
+	test.That(t, callCountNames, test.ShouldEqual, 6)
```

Comment:
I'm not entirely sure why `callCountSubtypes` and `callCountNames` are different. I thought they would be the same, but repeated runs of the tests show that these are the values.